opwen_email_server/mailers/wikipedia.py

```diff
@@ -0,0 +1,56 @@
+from base64 import b64encode
+from datetime import datetime
+from typing import Callable
+
+from requests import get
+from wikipedia import languages
+from wikipedia import page
+from wikipedia import set_lang
+from wikipedia.exceptions import DisambiguationError
+from wikipedia.exceptions import PageError
+
+from opwen_email_server.utils.log import LogMixin
+
+WIKIPEDIA_ADDRESS = 'wikipedia@bot.lokole.ca'
+
+
+class WikipediaEmailFormatter(LogMixin):
```

Comment:
Could you also add a unit test for this class? You can use the [responses](https://github.com/getsentry/responses) library to intercept the HTTP requests and return whatever content you need to return. There's several usage examples already in the tests folder.

Reply:
Ok, will do!